SPECS/tdnf/tdnf.spec

```diff
@@ -28,17 +31,22 @@ BuildRequires:  glibc-devel
 BuildRequires:  gpgme-devel
 BuildRequires:  libmetalink-devel
 BuildRequires:  libsolv-devel
+BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
```

Comment:
Should double check that the dependencies to build the new TDNF are satisfied in build_official_toolchain_rpms.sh.
Since this package is built in the toolchain environment, the BuildRequires in the spec are not parsed, and the toolchain packages build sequentially based on dependencies.
https://github.com/microsoft/CBL-Mariner/blob/main/toolkit/scripts/toolchain/build_official_toolchain_rpms.sh#L519

Reply:
Looks like everything that tdnf needs is still built/installed ahead of tdnf in this script.